src/Servers/Kestrel/Core/src/KestrelServerOptions.cs

```diff
@@ -87,6 +87,11 @@ public class KestrelServerOptions
         /// </summary>
         public bool EnableAltSvc { get; set; } = false;
 
+        /// <summary>
+        /// 
+        /// </summary>
+        public bool EnableWiresharkStyleLog { get; set; } = false;
```

Comment:
I'm not sure about using the name "WireShark" in public API. At the minimum, we'll have to run it by CELA to make sure there isn't a trademark issue here.

Is there a generic name for this log format?

cc @Pilchie

Reply:
We shouldn’t call it wire shark we can call it verbose or hex or something

Reply:
Do we need an option at all?

Reply:
True, we can just change it.

Reply:
It sounds like we're all in favor of just removing the option and always using the new format which need not be named.